src/changelog.rst

```diff
@@ -0,0 +1,190 @@
+
+.. caution::
+
+   This page is **not** an exhaustive list of all of the changes in a release,
+   just the language changes that had an impact to the FLS. See the `release
+   notes`_ for a full list of changes.
+
+Language changes in Rust 1.79.0
+-------------------------------
+
+* `Stabilize inline \`const {}\` expressions. <https://github.com/rust-lang/rust/pull/104087/>`_
+
+  * New section: :ref:`fls_G59PiNQkVUnQ`
+
+* `Prevent opaque types being instantiated twice with different regions within the same function. <https://github.com/rust-lang/rust/pull/116935/>`_
+
+  * No change: already described in :p:`fls_hza5n5eb18ta`
+
+* `Stabilize WebAssembly target features that are in phase 4 and 5. <https://github.com/rust-lang/rust/pull/117457/>`_
+
+  * No change: ``cfg`` and ``cfg_attr`` configuration predicates are not part of the FLS
+
+* `Add the \`redundant_lifetimes\` lint to detect lifetimes which are semantically redundant. <https://github.com/rust-lang/rust/pull/118391/>`_
+
+  * No change: lints are not part of the FLS
+
+* `Stabilize the \`unnameable_types\` lint for public types that can't be named. <https://github.com/rust-lang/rust/pull/120144/>`_
+
+  * No change: lints are not part of the FLS
+
+* `Enable debuginfo in macros, and stabilize \`-C collapse-macro-debuginfo\` and \`#[collapse_debuginfo]\`. <https://github.com/rust-lang/rust/pull/120845/>`_
+
+  * New section: :ref:`fls_qyudjGHZfyJH`
+
+* `Propagate temporary lifetime extension into \`if\` and \`match\` expressions. <https://github.com/rust-lang/rust/pull/121346/>`_
+
+  * New paragraphs: :p:`fls_Rj9zhVutfQod`, :p:`fls_oodpp3LpXC13`, :p:`fls_xGThCPoTUSAi`
+
+* `Restrict promotion of \`const fn\` calls. <https://github.com/rust-lang/rust/pull/121557/>`_
+
+  * No change: already described in :p:`fls_3h5vr7xk2rrt`
+
+* `Warn against refining impls of crate-private traits with \`refining_impl_trait\` lint. <https://github.com/rust-lang/rust/pull/121720/>`_
+
+  * No change: lints are not part of the FLS
+
+* `Stabilize associated type bounds (RFC 2289). <https://github.com/rust-lang/rust/pull/122055/>`_
+
+  * New paragraph: :p:`fls_mcUMWsYcxzmZ`
+
+* `Stabilize importing \`main\` from other modules or crates. <https://github.com/rust-lang/rust/pull/122060/>`_
+
+  * No change: this lifted restriction was not previously described in the FLS
+
+  * While updating the FLS to account for this feature, we realized that the
+    way the FLS described crate types was incorrect. We rectified this:
+
+    * New section: :ref:`fls_8JB3SJqamdpU`
+    * New glossary entry: :t:`crate type`
+    * New paragraphs: :p:`fls_unxalgMqIr3v`, :p:`fls_e7jGvXvTsFpC`, :p:`fls_kQiJPwb2Hjcc`, :p:`fls_OyFwBtDGVimT`
+    * Updated glossary entries: :t:`binary crate`, :t:`proc-macro crate`
+    * Updated paragraphs: :p:`fls_9ub6ks8qrang`, :p:`fls_Mf62VqAhoZ3c`
+    * Moved paragraph: :p:`fls_sbGnkm8Ephiu`
+    * Removed paragraph about library crates
```

Comment:
Re-reading the change to reword this section, it feels wrong that we removed the description of library crates in the FLS. Can you send a follow-up PR to add back that paragraph and glossary entry?